.travis.yml

```diff
@@ -20,10 +20,13 @@ php:
 matrix:
     include:
         - php: 7.2
-          env: COMPOSER_FLAGS='--prefer-lowest --prefer-stable' SYMFONY_DEPRECATIONS_HELPER=weak
+          env:
+              - COMPOSER_FLAGS='--prefer-lowest --prefer-stable'
+              - SYMFONY_DEPRECATIONS_HELPER=weak
         - php: 7.3
-          env: DEPENDENCIES='dev'
-          env: SYMFONY_DEPRECATIONS_HELPER='max[self]=0'
+          env:
+              - DEPENDENCIES='dev'
+              - SYMFONY_DEPRECATIONS_HELPER='max[self]=0'
         # Test LTS version
         - php: 7.3
           env: SYMFONY_VERSION='^3'
```

Comment:
I guess we should add Symfony 5 to our matrix (and maybe use DEPENDENCIES dev on that)
Incidentally, we're missing Symfony 4 too

Reply:
Also, the version of PHPUnit used by `simple-phpunit` had to be pinned at 7.5. Travis didn't report a fail, but pretty sure we don't want [this](https://travis-ci.org/KnpLabs/KnpMenuBundle/jobs/614518051)